### PR TITLE
LPS-173447 - Add theme spritemap to sample workspace

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

File: workspaces/refresh_liferay_sample_workspace.sh [hunk 1]
```diff
@@ -227,6 +227,126 @@ EOF
 	../tools/create_custom_element.sh liferay-sample-custom-element-3 angular
 
 	mv liferay-sample-custom-element-3 liferay-sample-workspace/client-extensions
+
```
Comment (izaera) [inline workspaces/refresh_liferay_sample_workspace.sh:230]: So, what's this doing? :flushed: 

If the files are added explicitly to the sample workspace, why is this script needed and for what? :thinking:

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
manually forwarded at https://github.com/brianchandotcom/liferay-portal/pull/132303#issuecomment-1482714836

**Pull request closed without merging**